config.example.json

```diff
@@ -1,7 +1,9 @@
 {
   "Runtime": {
     "DryRunMode": false,
-    "Verbosity": 1
+    "Verbosity": 1,
+    "SentryDSN": "",
+    "Environment": "production"
```

Comment:
Different than the type definition, but same as the other example. Which makes me wonder why we have two examples.

Reply:
Yeah, I was thinking that too. Do you have a preference which to keep? They both are somewhat fitting. I'd lean towards keeping the lambda-example config though.